scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
Hmm that sucks! You can hack your way around this. Just run your postdeploy script in your `heroku_startup` script instead and make sure you only run it based on the env (which you should override for staging and production on heroku itself). It would take some testing to make sure that you're still within the 60s time limit to bind to the port.